src/python/pants/backend/python/subsystems/pytest_v2.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from typing import Tuple
+
+from pants.subsystem.subsystem import Subsystem
+
+
+class PytestV2(Subsystem):
+  options_scope = 'pytestv2'
+
+  @classmethod
+  def register_options(cls, register):
+    super().register_options(register)
+    register('--version', default='pytest>=5.2.2', help="Requirement string for Pytest.")
+    register(
+      '--pytest-plugins',
```

Comment:
NB: this, unfortunately, cannot be `plugins` because that is taken by the global namespace :/